index.html

```diff
@@ -5,6 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: red;
```

Comment:
These colors might pose an accessibility problem for some end users

Reply:
Consider instead
```
background-color: pink
```